src/Compilers/CSharp/Portable/Symbols/Attributes/WellKnownAttributeData/ParameterWellKnownAttributeData.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis
```

Comment:
`Microsoft.CodeAnalysis.CSharp.Symbols`

Reply:
`CSharp.Symbols`

---
In reply to: [215753507](https://github.com/dotnet/roslyn/pull/29636#discussion_r215753507) [](ancestors = 215753507)